app/repositories/SessionRepository.scala

```diff
@@ -135,12 +139,12 @@ class DefaultSessionRepository @Inject() (config: AppConfig, component: Reactive
       Left(DatabaseError)
     }
 
-  def get(key: String, pageHistoryUrl: Option[String], previousPageByLink: Boolean): Future[RequestOutcome[ProcessContext]] =
+  def get(key: String, pageUrl: Option[String], previousPageByLink: Boolean): Future[RequestOutcome[ProcessContext]] =
```

Comment:
If you are changing the name here it might be worth changing the name in the calling method in the guidance service also.

Reply:
done